insteon_mqtt/handler/Broadcast.py

```diff
@@ -28,7 +28,10 @@ class Broadcast(Base):
     Finally a broadcast LINK_CLEANUP_REPORT is sent.  This message indicates
     if the device received ACKs from all linked devices or not.  This message
     indicates that the device is finished sending messages.  However, as
-    broadcast message, it is not guaranteed to be received.
+    broadcast message, it is not guaranteed to be received.  Some devices
+    even have a user option to turn off these messages.  These messages
+    contain information about how the success or failure of the broadcast but
```

Comment:
This feels like awkward wording.  Maybe the word "how" doesn't belong here?
```suggestion
    contain information about the success or failure of the broadcast but
```